frame_body: os_sensor
sensors:
    -
        topic: /d455/depth/image_rect_raw
        frame: d455_depth_optical_frame
        cam_info_topic: /d455/depth/camera_info
        bins_w: 16
        bins_h: 9
        percentile: 10
        min_ratio_per_bin: 0  # [0-1]
        mm_resolution: 1
        min_range: 0.1  # [m]
        max_range: 10  # [m]
        is_pointcloud: false
        is_polar: false
    -
        topic: /os_cloud_node/range_image
        frame: os_sensor
        bins_w: 32
        bins_h: 8
        percentile: 10  # [%]
        min_ratio_per_bin: 0  # [0-1]
        mm_resolution: 4  # mm resolution in 1 pixel unit
        min_range: 0.6  # [m]
        max_range: 10  # [m]
        is_pointcloud: false
        is_polar: true
        hfov: 360
        vfov: 90
        image_w: 512
        image_h: 128
